recipe/meta.yaml

```diff
@@ -1,59 +1,83 @@
-{% set version = "0.8.2" %}
+{% set version = "0.8.5" %}
 
 package:
   name: nbgrader
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/n/nbgrader/nbgrader-{{ version }}.tar.gz
-  sha256: d1a375ccdce31b3bd45a1110b525e4fc3b9e21ad916a22a1e682b147311ed425
+  sha256: 22ea4a53cf77fccbe4aea10196782040a4d9f357730a463b0eb3db368311dea4
 
 build:
   noarch: python
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - nbgrader = nbgrader.apps.nbgraderapp:main
 
 requirements:
   host:
-    - python >=3.8
-    - pip
-    - hatchling >=1.10.0
     - hatch-jupyter-builder >=0.7
-    - jupyterlab <4
+    - hatchling >=1.10.0
+    - jupyterlab >=3.1,<4
+    - pip
+    - python >=3.7
```

Comment:
This is an error in `nbgrader` repo since IPython>=8.0.0 does not support Python<3.8.
I'll fix it, but it can be updated here.

Reply:
`ipython`'s pin will be effective here, so it's unlikely to cause an issue... would just wait until https://github.com/jupyter/nbgrader/blob/0.8.x/pyproject.toml#L12 is accurate before doing anything else here.

Reply:
Should we also release a new version ?

Reply:
I mean, python 3.7 isn't supported anymore, and unless 0.8.5 was yanked from pypi, 3.7 users might still suffer some confusion. We just try to be _accurate_ unless something is reported as _actually_ broken.

Reply:
0f884c2